tests/Database/DatabaseFunctionalTest.php

```diff
@@ -100,6 +101,22 @@ public function testCommand()
         $this->assertTrue($commandResult->ismaster);
     }
 
+    public function testCommandDoesNotInheritReadPreference()
+    {
+        if ($this->getPrimaryServer()->getType() !== Server::TYPE_RS_PRIMARY) {
+            $this->markTestSkipped('Test only applies to replica sets');
+        }
+
+        $this->database = new Database($this->manager, $this->getDatabaseName(), ['readPreference' => new ReadPreference(ReadPreference::RP_SECONDARY)]);
```

Comment:
I like that this test should also pass even if the replica set has a single primary member :+1: